LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2016-2017 Rob Wu rob@robwu.nl
```

Comment:
I guess you copied the rendered markdown? If you want the actual source via Github's interface, click on the "Raw" button/link and you will see the original source.